### DISCUSSION TITLE
There is no my-pages branch in the Publish Packages skill

### DISCUSSION THREAD

**Discussion by Rod-at-DOH** [skills/.github#213] [Q&A] [answered]
I'm going through the Publish Package skill. In Step 3, in the second, optional step it says:

> 2. (optional) Delete the branch `my-pages`.

There is no branch named `my-pages`.

**Comment by sinsukehlab** [comment, marked as answer]
This commit has accidentally changed the branch name.
skills/publish-packages@7e3731f (skills/publish-packages#18)

**Reply by sinsukehlab** [reply]
You can open a PR to fix this.

**Reply by Rod-at-DOH** [reply]
Hi @sinsukehlab, I've made the modification to my fork of the original Skills repo. I've pushed it to my GH repo of `publish-packages`. So, then I tried looking up how to create a PR back to the `skills/publish-packages`, but of the two resources I found [How to make your first pull request on GitHub](https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/) and [How to Submit a Pull Request to Someone Else's Repository](https://learntocodetogether.com/create-your-first-pull-request/), they both show using a button on the Code tab labeled, "Clone or Download". I do not have that in my repo for `publish-packages`.

I need additional help with this, please.

**Reply by Rod-at-DOH** [reply]
@sinsukehlab I've resolved the issue. And I've created a PR.

**Reply by sinsukehlab** [reply]
Yes, you should have generated a repo with a different name from the template. Forked repos usually have the same name with the original.

**Reply by Rod-at-DOH** [reply]
Ah, I was wondering why I was having some many troubles. Thank you for that information!

**Reply by sinsukehlab** [reply]
I'm glad I could be of your help.
